lib/sb3_schema.json

```diff
@@ -29,7 +41,7 @@
                 },
                 "dataFormat": {
                     "type": "string",
-                    "enum": ["png", "PNG", "svg", "SVG", "jpeg", "JPEG", "jpg", "JPG", "bmp", "BMP"]
+                    "enum": ["png", "PNG", "svg", "SVG", "jpeg", "JPEG", "jpg", "JPG", "bmp", "BMP", "gif", "GIF"]
```

Comment:
Is it possible to set these to ignore capitalization? Or maybe something like:
```
"pattern": ["[pP][nN][gG]", "[sS][vV][gG]", ...
```

It's not a big deal, but I have on occasion seen files with extensions like "Jpeg." Maybe we should assume that the editor (GUI or whatever) will fix up the file name before saving, though... in which case maybe we should just list out the lower-case versions?

Reply:
Good point! I think I'll change it to only accept lower case and make sure that we're only serializing lowercase.